.github/driver.sh

```diff
@@ -22,9 +22,9 @@ unset CARGO_MANIFEST_DIR
 
 # Run a lint and make sure it produces the expected output. It's also expected to exit with code 1
 # FIXME: How to match the clippy invocation in compile-test.rs?
-./target/debug/clippy-driver -Dwarnings -Aunused -Zui-testing --emit metadata --crate-type bin tests/ui/cstring.rs 2> cstring.stderr && exit 1
-sed -e "s,tests/ui,\$DIR," -e "/= help/d" cstring.stderr > normalized.stderr
-diff normalized.stderr tests/ui/cstring.stderr
+./target/debug/clippy-driver -Dwarnings -Aunused -Zui-testing --emit metadata --crate-type bin tests/ui/cast.rs 2> cast.stderr && exit 1
+sed -e "s,tests/ui,\$DIR," -e "/= help/d" cast.stderr > normalized.stderr
+diff normalized.stderr tests/ui/cast.stderr
```

Comment:
can you change the `diff` call to `diff -u ...`?
This should make things more human readable (it'll be in the same format `git diff` that way)

Reply:
For example, if I use `double_neg.stderr` not `cast.stderr`, the test seems to succeed. I still don't know if the test here should be `cast.stderr`.

Reply:
Looking at https://github.com/rust-lang/rust-clippy/pull/6257/commits/a1bb10e9b805d3bd60d7c47dc453c011478245df, I didn't know if it had to be `cast.stderr` in the test here, so I use `double_neg.stderr`. If it's not just testing in one file and  it has to be `cast.stderr`,  I would consider modifying the sed regex.